src/lib/libraries/costumes.json

```diff
@@ -3003,17 +3003,6 @@
             2
         ]
     },
-    {
```

Comment:
Just to double check, is this gone intentionally?

Reply:
yup- it's a duplicate